core/src/main.rs

```diff
@@ -98,37 +103,66 @@ fn main() {
         })
         .ok();
 
-    let op = OraclePool::new().unwrap();
-    loop {
-        if let Err(e) = main_loop_iteration(args.read_only, &op) {
-            error!("Fatal error: {:?}", e);
-            std::process::exit(exitcode::SOFTWARE);
+    match args.command {
+        Command::Bootstrap { yaml_config_name } => {
+            let wallet = WalletData {};
+            if let Err(e) = (|| -> Result<(), anyhow::Error> {
+                let _ = bootstrap::bootstrap(
+                    yaml_config_name,
+                    &wallet,
+                    &wallet,
+                    &wallet,
+                    get_change_address_from_node()?,
+                    current_block_height()? as u32,
+                    0,
```

Comment:
When starting up a new pool, is there a reasonable initial datapoint value? The pool box needs it.